specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2697,6 +3231,230 @@
       ],
       "description": "Parameter required for creating a linked server to redis cache."
     },
+    "RedisCacheAccessPolicy": {
+      "description": "Base properties for an access policy",
+      "type": "object",
+      "properties": {
+        "commandPermissions": {
+          "type": "string",
+          "description": "Command permissions for the access policy."
+        },
+        "keyPermissions": {
+          "type": "string",
+          "description": "Key permissions for the access policy."
+        },
+        "pubSubChannelPermissions": {
+          "type": "string",
+          "description": "Pub sub channel permissions for the access policy."
+        }
+      },
+      "required": [
+        "commandPermissions",
+        "keyPermissions",
+        "pubSubChannelPermissions"
```

Comment:
Why have all of them be required? Wouldn't it be reasonable to specify or use just one type of permissions?

Reply:
We're separating permissions to make it easier for validation. Removing keyPermissions and pubSubChannelPermisssions from required, only commandPermissions will be mandatory now.

Reply:
Changed to only one field for permissions now